cpu/sam0_common/periph/eth.c

```diff
@@ -0,0 +1,315 @@
+/*
+ * Copyright (C) 2020 Mesotic SAS
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     cpu_sam0_common
+ * @{
+ *
+ * @file
+ * @brief       Low-level Ethernet driver implementation
+ *
+ * @author      Dylan Laduranty <dylan.laduranty@mesotic>
```

Comment:
Missing top level domain in email address?

Reply:
Fixed.